src/markdown/faq.md

```diff
@@ -126,7 +126,7 @@ Indeed you can have 2 main CSS classes at the same DOM node. In BEM we call it `
 <div class="block1 block2"></div>
 ```
 
-But important thing about it is that both `block1` and `block2` should be standalone blocks. This is slightly different from what people usually mean by "global modifiers" as modifiers do not have any sense on they own and are just a set of properties to change.
+But the important thing about it is that both `block1` and `block2` should be standalone blocks. This is slightly different from what people usually mean by "global modifiers," as modifiers do not have any sense on their own and are just a set of properties to change.
```

Comment:
Is `,` supposed to be inside quotes?

Reply:
That's the rule I learned in American primary (i.e. elementary) school... **but** I have disliked that rule ever since I began reading and writing software documentation.

According to [this article on the APA website](http://blog.apastyle.org/apastyle/2011/08/punctuating-around-quotation-marks.html) (screenshot below), that rule is part of the so-called "American Style" of punctuation.

> ![image](https://user-images.githubusercontent.com/7143133/32696376-2729cebe-c72b-11e7-979d-f9e42284fa99.png)


This is the first time I've researched the rule. I'm happy to see there is an alternative style (i.e. the so-called "British Style") that dictates that the comma be *outside* the quotation marks. I think putting the comma outside the quotation marks would make for a less ambiguous statement.

I'll update the PR to use the "British Style."

Reply:
Done in https://github.com/getbem/getbem.com/pull/149/commits/ed7f2a104a61c920526080d8524facc6355e1cd9